netbox/netbox/configuration_example.py

```diff
@@ -221,6 +221,11 @@
 # database access.) Note that the user as which NetBox runs must have read and write permissions to this path.
 SESSION_FILE_PATH = None
 
+# By default the memory and disk sizes are displayed using base 10 (e.g. 1000 MB = 1 GB).
+# If you would like to use base 2 (e.g. 1024 MB = 1 GB) set this to 1024.
+DISK_BASE_UNIT = 1024
```

Comment:
would want these commented out as if you copied this over the default would change.